assets/css/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,81 @@
+// Based on https://scottaohara.github.io/a11y_styled_form_controls/src/checkbox--switch/
```

Comment:
Instead of a code comment, thoughts on tossing this in the commit message? I find that code comments easy fall out of date over time, but a commit message is tied to this particular change when we leveraged Scott’s work.

Reply:
Makes sense! I'll include the link to `https://scottaohara.github.io/a11y_styled_form_controls/src/checkbox--switch/` in the squash-merge commit description.